packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -1088,7 +1088,7 @@ const AnalyticalTable = forwardRef((props: AnalyticalTablePropTypes, ref: Ref<HT
         )}
       </div>
       <Text aria-hidden="true" id={`smartScaleModeHelper-${uniqueId}`} className={classes.hiddenSmartColMeasure}>
-        ""
+        {''}
```

Comment:
I think we can just self-close this `Text`, so no need for an empty string to be rendered.

Reply:
children are mandatory